app/javascript/components/molecules/Popover.js

```diff
@@ -17,23 +18,21 @@ const useClickOutside = (ref, callback) => {
   })
 }
 
-const Popover = ({ buttonElement, children }) => {
+const Popover = ({ buttonElement, children, onButtonClick }) => {
```

Comment:
nit: can you set defaults for non-required props here? Even if it's just onButtonClick = null